coral/src/app/router.tsx

```diff
@@ -18,6 +19,10 @@ const routes: Array<RouteObject> = [
     path: "/topics/request",
     element: <RequestTopic />,
   },
+  {
+    path: "/topic/:topicName/acl/request",
```

Comment:
This is the old pattern, will be updated when we start work on https://github.com/aiven/klaw/issues/452